documentation/modules/ROOT/pages/_partials/compile-and-run.adoc

```diff
@@ -45,12 +83,30 @@ To run the application in native mode
 Native container-build mode::
 +
 --
-[#basics-build-quarkus-native-docker-app]
+[%header,cols="1,1"]
+|=====
+
+|Maven
+
+|Quarkus CLI
+
+a|
+[#basics-build-quarkus-native-docker-app-mvn]
+[.console-input]
+[source,bash,subs="+macros,+attributes"]
+----
+mvn package -Pnative \
+-Dquarkus.native.container-build=true 
+----
+a|
+[#basics-build-quarkus-native-docker-app-quarkus]
 [.console-input]
 [source,bash,subs="+macros,+attributes"]
 ----
-mvn package -DskipTests -Pnative -Dquarkus.native.container-build=true
+quarkus build --native \
+ -Dquarkus.native.container-build=true
```

Comment:
If we update the tutorial to Quarkus 2.15 quarkus CLI offers ways to build containers. https://quarkus.io/blog/quarkus-2-15-0-final-released/#cli-improvements

Reply:
That's what I wanted to do, but I ran into a bug with this.  I created a pull request in the Quarkus project to fix this (https://github.com/quarkusio/quarkus/pull/30194)

Reply:
ok nevermind, I just need to make sure I use a new project where they've added a -Dnative property (instead of the previous -Pnative)